client/client.go

```diff
@@ -26,9 +26,6 @@ import (
 	"google.golang.org/grpc"
 	"google.golang.org/grpc/credentials"
 
-	// allow multi gRPC URLs
-	//_ "github.com/Jille/grpc-multi-resolver"
```

Comment:
are you sure we don't need any initialization to happen here?

Reply:
The code was already commented out, was just cleaning it up. 

I'm guessing, even if the init is required, it should be part of another PR ?

Reply:
oh, let it be as it is then :)

Reply:
Have reverted my change